RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,864 @@
+        "412":
+          description: The client attempted to transition the operational intent to Accepted or Activated while marked as Down in the DSS.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "413":
+          description: The area of the operational intent is too large.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "429":
+          description: The client issued too many requests in a short period of time.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+      security:
+      - Authority:
+        - utm.strategic_coordination
+
+components:
+  schemas:
+    ResourceShape:
+      description: The physical shape of the resource.  This is not the same as the resource extent saved in the DSS. 
+      anyOf:
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Circle'
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/LatLngPoint'
+        - $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Polygon'
+    ResourcePhysicalDefinition:
+      required:
+        - name
+        - shape
+        - type
+      properties:
+        name:
+            description: Name of the resource
+            example: CHIPR
+            $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/TextShortType'
+        shape:
+          $ref: '#/components/schemas/ResourceShape'
+        altitude_lower:
+          description: Lower altitude limit of the resource, if any. 
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        altitude_upper:
+          description: Upper altitude limit of the resource, if any. 
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        type:
+          $ref: '#/components/schemas/ResourceType' 
+    ResourceStatus:
+      required: 
+        - status
+      properties:
+        status: 
+          type: string
+          description: The current status of the resource. 
+          enum: 
+          - ACTIVE
+          - INACTIVE
+    ResourceType:
+      required: 
+        - resource_type
+      properties:
+        resource_type: 
+          type: string
+          description: The type of resource.  
+          enum: 
+          - WAYPOINT
+          - VOLUME
+    CapacityLimitType:
+      required:
+        - capacity_type
+      type: object
+      properties:
+        capacity_type:
+          type: string
+          description: |
+            The type of capacity limitation which is to be met for the resource. 
+          externalDocs:
+            url: TBD
+          enum: 
+          - FLOW_RATE_CONSTANT
```

Comment:
FLOW_RATE_CONSTANT is unclear to me. Is this intended to describe a type where the start time of the bin is not fixed, and the constraint remains valid as the bin moves along the timeline? If so, we intended to use FLOW_RATE_ROLLING_BIN.